### PR TITLE
fixed move More... down issue Fixes #332, Fixes #333

### LINKED ISSUE #332: Move more... link to bottom on ODM Custom posts widget
home-2

### LINKED ISSUE #333: Reduce spacing between ODM Custom posts widget
home-2

### PR DESCRIPTION
**Pull request by S-mardii**
The change result in this:

screenshot 2

### CODE REVIEW

**Comment by S-mardii** [comment]
"More..." is not on the same row

screenshot 2

File: inc/widgets/odm-custom-posts-widget.php [hunk 1]
```diff
@@ -93,7 +91,9 @@ public function widget( $args, $instance ) {
 						$index++;
 					endforeach;
 					?>
-
+				<div class="sixteen columns align-right">
```
Comment (acorbi) [inline inc/widgets/odm-custom-posts-widget.php]: @S-mardii here, you are adding ```<div class="sixteen columns align-right">``` inside another ```<div class="sixteen columns">```. I would suggest you to not nest them but put them at the same level

```
<div class="sixteen columns">
</div>
 <div class="sixteen columns align-right"> 
</div>
```

**Comment by S-mardii** [comment]
@acorbi I forgot to inform that this pull request also intent to fix issue #333

File: inc/widgets/odm-custom-posts-widget.php (hunk 1 above)
Comment (S-mardii) [inline inc/widgets/odm-custom-posts-widget.php]: I will check on that

**Comment by acorbi** [comment]
@S-mardii noted. however it is better if you create 1 PR for each Issue. Let's do like that the next time ;-)

**Pull request merged**